### PR TITLE
Fixes #2 grammar error correction, add sample responses to REST API

### LINKED ISSUE #2: grammar error correction, add sample responses to REST API
Small changes related to readability.

### PR DESCRIPTION
**Pull request by kkasravi**

### CODE REVIEW

**Comment by kkasravi** [comment]
I'm not getting any metrics returned for 

```
curl http://127.0.0.1:8090/metrics/app/1/app1.processor0
```

File: docs/0.3/restapi.md [hunk 1]
```diff
@@ -18,13 +32,44 @@ Example:
...
+```
 
 ## GET config/app/&lt;appId&gt;
 Query the configuration of specific application appId
 
 Example:
 ```bash
-curl http://127.0.0.1:8090/config/app/0
+curl http://127.0.0.1:8090/appmaster/1
+```
```
Comment (clockfly) [inline docs/0.3/restapi.md:59]: This is incorrect, should be config/app/appId?

**Comment by clockfly** [comment]
I can double check the metrics

File: docs/0.3/restapi.md (hunk 1 above)
Comment (kkasravi) [inline docs/0.3/restapi.md:59]: Let me try again. I tried that and didn't get anything back.

Comment (clockfly) [inline docs/0.3/restapi.md:59]: Yes, it will return an empty map if you have set the ClusterConfigSource in AppDescription.

**Comment by clockfly** [comment]
About the metrics, it should work. However, it will requre 5 seconds before you get any metrics data.

As in the backend, we have a metrics report interval setting.

**Comment by clockfly** [comment]
Let me fix the rest for you

**Comment by kkasravi** [comment]
Thanks Sean
